installing/installing_alibaba/installing-alibaba-customizations.adoc

```diff
@@ -0,0 +1,60 @@
+[id="installing-alibaba-customizations"]
+= Installing a cluster on Alibaba with customizations
+include::modules/common-attributes.adoc[]
+:context: installing-alibaba-customizations
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a customized cluster on infrastructure that the installation program provisions on Alibaba Cloud. To customize the installation, you modify
+parameters in the `install-config.yaml` file before you install the cluster.
+
+[NOTE]
+====
+The scope of the {product-title} installation configurations is intentionally narrow. It is designed for simplicity and ensured success. You can complete many more {product-title} configuration tasks after an installation completes.
+====
+
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_alibaba/installing-alibaba-account.adoc#installing-alibaba-account[configured an Alibaba Cloud account] to host the cluster.
+* If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can xref:../../installing/installing_alibaba/manually-creating-iam-alibaba.adoc#manually-creating-iam-alibaba[manually create and maintain IAM credentials].
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/ssh-agent-using.adoc[leveloffset=+1]
+
+include::modules/installation-obtaining-installer.adoc[leveloffset=+1]
+
+include::modules/installation-initializing.adoc[leveloffset=+1]
+
+include::modules/installation-configuration-parameters.adoc[leveloffset=+2]
+
+include::modules/installation-aws-config-yaml.adoc[leveloffset=+2]
```

Comment:
Need to create one section for `alibabacloud`. More information about the options can be found on the install-config schema: https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L20

Not sure if we have another better place to get all valid options. :) 
cc @staebler

Reply:
FYI generally QE would refer to "openshift-install explain".

$ **openshift-install explain installconfig.platform.alibabacloud**                                                
KIND:     InstallConfig                                                                                                                 
VERSION:  v1                                                                                                                            
                                                                                                                                        
RESOURCE: <object>                                                                                                                      
  AlibabaCloud is the configuration used when installing on Alibaba Cloud.                                                              
                                                                                                                                        
FIELDS:
    defaultMachinePlatform <object>
      DefaultMachinePlatform is the default configuration used when installing on Alibaba Cloud for machine pools which do not define their own platform configuration.

    privateZoneID <string>
      PrivateZoneID is the ID of an existing private zone into which to add DNS records for the cluster's internal API. An existing private zone can only be used when also using existing VPC. The private zone must be associated with the VPC containing the subnets. Leave the private zone unset to have the installer create the private zone on your behalf.

    region <string> -required-
      Region specifies the Alibaba Cloud region where the cluster will be created.

    resourceGroupID <string>
      ResourceGroupID is the ID of an already existing resource group where the cluster should be installed. If empty, the installer will create a new resource group for the cluster.

    tags <object>
      Tags additional keys and values that the installer will add as tags to all resources that it creates. Resources created by the cluster itself may not include these tags.

    vpcID <string>
      VpcID is the ID of an already existing VPC where the cluster should be installed. If empty, the installer will create a new VPC for the cluster.

    vswitchIDs <[]string>
      VSwitchIDs is the ID list of already existing VSwitches where cluster resources will be created. The existing VSwitches can only be used when also using existing VPC. If empty, the installer will create new VSwitches for the cluster.

$ **openshift-install explain installconfig.platform.alibabacloud.defaultMachinePlatform**
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <object>
  DefaultMachinePlatform is the default configuration used when installing on Alibaba Cloud for machine pools which do not define their own platform configuration.

FIELDS:
    imageID <string>
      ImageID is the Image ID that should be used to create ECS instance. If set, the ImageID should belong to the same region as the cluster.

    instanceType <string>
      InstanceType defines the ECS instance type. eg. ecs.g6.large

    systemDiskCategory <string>
      Valid Values: "","cloud_efficiency","cloud_essd"
      SystemDiskCategory defines the category of the system disk.

    systemDiskSize <integer>
      SystemDiskSize defines the size of the system disk in gibibytes (GiB).

    zones <[]string>
      Zones is list of availability zones that can be used. eg. ["cn-hangzhou-i", "cn-hangzhou-h", "cn-hangzhou-j"]

$